CONTRIBUTING.md

```diff
@@ -18,10 +18,11 @@
     commit message, wrapped at 72 characters. Do not commit changes to
     `dist/ramda.js`.
 
-1.  Run `npm test` (or `make test lint`) and address any errors. It will install
```

Comment:
Side note: I looked at the makefile and it doesn't look like it is linting either. I am not super experienced with `make` though so I don't really want to update it.

Reply:
I think we need to spend some real time working through our build.  What are the goals?  What do we use to reach them?  How can we do so consistently?

Right now this part has grown very haphazardly.  I think it's time to it under control.